<launch> 
    <node name="engine" pkg="adaptation_engine" type="engine" output="screen" />

    <!--<param name="frequency" value="1" /> --> <!--1 Hz the module frequency is the same as the monitors--> 
    <param name="monitor_freq" value="10" />        <!-- Hz -->

    <param name="setpoint" value="0.95" />          <!-- -->
    <param name="actuation_freq" value="0.02" />    <!-- Hz -->
    <param name="info_quant" value="500" />         <!-- [1,1000] -->

    <param name="offset" value="0.7" />            <!-- % of the current state | Original = 0.55-->
    <param name="gain" value="0.03" />              <!-- search granularity | Original = 0.0005-->

</launch>
